frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
- [ ] When I create appointments and login with another user, the same appointments show up, as if I was still the first user.

Kindly correct this bug, so your app works smoothly as intended. 💪